src/linter_test.cc

```diff
@@ -91,5 +91,19 @@ TEST(LinterTest, AliasKeywordCheck) {
     EXPECT_TRUE(CheckAliasKeyword("SELECT 1 AS one").ok());
 }
 
+TEST(LinterTest, TabCharactersUniformCheck) {
```

Comment:
Can you also add test cases for negative cases which also check the error position reported? E.g.

EXPECT_FALSE(CheckTabCharactersUniform("  SELECT 5;\n  \tSELECT 6;").ok());